sdk/eventhub/tests.yml

```diff
@@ -5,6 +5,7 @@ jobs:
     parameters:
       AllocateResourceGroup: 'false'
       ServiceDirectory: eventhub
+      InjectedPackages: $(WheelLocationsList)
```

Comment:
MARK: this is for testing unreleased uamqp v1.2.14 wheels only, will be removed after uamqp 1.2.14 is released